packages/cysync-core/src/pages/OnBoarding/SetPassword/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { LogoOutlinedAsideImage, SuccessDialog } from '@cypherock/cysync-ui';
+
+import { routes } from '~/constants';
+import { PasswordForm } from './Dialogs/PasswordForm';
+import { selectLanguage, useAppSelector } from '../../../store';
+import { OnboardingPageLayout } from '../OnboardingPageLayout';
+
+export const SetPassword: React.FC = () => {
+  const [isPasswordSet, setIsPasswordSet] = React.useState(false);
```

Comment:
Use `useLockscreen` hook

Reply:
This is just for local UI update not needed here